frontend/src/__mocks__/mockInferenceServiceK8sResource.ts

```diff
@@ -0,0 +1,87 @@
+import { InferenceServiceKind } from '~/k8sTypes';
+
+type mockResourceConfigType = {
```

Comment:
PascalCase please...
```suggestion
type MockResourceConfigType = {
```